contracts/prize-pool/PrizePool.sol

```diff
@@ -828,6 +837,14 @@ abstract contract PrizePool is OwnableUpgradeSafe, RelayRecipient, ReentrancyGua
     return earlyExitFee;
   }
 
+  /// @notice Allows the Governor to set a cap on the amount of liquidity that he pool can hold
+  /// @param _liquidityCap The new liquidity cap for the prize pool
+  function setLiquidityCap(uint256 _liquidityCap) external onlyOwner {
+    require(_liquidityCap == 0 || _tokenTotalSupply() <= _liquidityCap, "PrizePool/supply-exceeds-cap");
```

Comment:
Let's not have any special values!  Let's let 0 be zero- no deposits allowed.  The liquidity cap can be initialized in the constructor to the max int

Reply:
Also- it doesn't matter if the cap is less than the currently supply of tokens.  That kind of the point- it'll force withdrawals until liquidity drops below the cap at which point deposits can resume